packages/blade/src/utils/storybook/StoryPageWrapper.tsx

```diff
@@ -116,6 +125,22 @@ const StoryPageWrapper = (props: StoryPageWrapperTypes): React.ReactElement => {
             <>
               <BaseBox id="properties-ref">
                 <Title>Properties</Title>
+                {props.apiDecisionLink === '' ? null : (
+                  <Text marginY="spacing.5">
+                    You can checkout API Decisions for {props.componentName} at{' '}
+                    <Link
+                      target="_blank"
+                      href={
+                        props.apiDecisionLink ??
+                        `https://github.com/razorpay/blade/blob/master/packages/blade/src/components/${props.componentName}/_decisions/decisions.md`
+                      }
+                    >
+                      {trimmedAPIDecisionLink
+                        ? trimmedAPIDecisionLink
+                        : `/blade/src/components/${props.componentName}/_decisions/decisions.md`}
```

Comment:
Let's put the Component's name instead of the API decision path?

Reply:
eg: 

You can checkout API Decisions for [Button here](https://github.com/razorpay/blade/blob/master/packages/blade/src/components/Button/_decisions/decisions.md)

Reply:
<img width="293" alt="image" src="https://github.com/razorpay/blade/assets/30949385/52322b81-4a3c-4383-b7f4-e05ef44add87">

done